src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -84,7 +91,25 @@ public void initialized() {
         initialized = true;
 
         if (gerritPushTargetPanels.size() == 1) {
-            branchTextField.setText(gerritPushTargetPanels.values().iterator().next());
+            String branchName;
+
+            try {
+                String gitReviewFilePath = StringUtils.join(
+                    new String[]{ProjectManager.getInstance().getOpenProjects()[0].getBasePath(), GITREVIEW_FILENAME}, File.separator);
+
+                Properties properties = new Properties();
+                properties.load(new FileInputStream(gitReviewFilePath));
+
+                branchName = properties.getProperty("defaultbranch");
+            } catch (IOException e) {
```

Comment:
I think checking if the file exists would be a bit nicer instead of catching the exception. Also, the catch block could be empty, because you do the branchName assignment anyway when it is empty a few lines below.

Please extract this reading code into a new method.

Reply:
Thank you for reviewing my PR. Implemented all your suggestions.